src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -121,6 +123,15 @@ private CallbackResponse handleAdmitPartOfClaim(CallbackParams callbackParams) {
                 .errors(errors)
                 .build();
         }
+        if (caseData.getRespondToAdmittedClaimOwingAmount() != null) {
+            BigDecimal valuePounds = caseData.getRespondToAdmittedClaimOwingAmount().divide(
```

Comment:
Is it possible to use function penniesToPounds in MonetaryConversions class in order to change from pennies to pounds?